src/components/shared/CopyableAddressV2/CopyableAddressV2.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback } from 'react';
+import clsx from 'clsx';
+import { splitAddress } from '~utils/strings';
+
+import { Address } from '~types';
+
+import MaskedAddress from '~shared/MaskedAddress';
+import useClipboardCopy from '~hooks/useClipboardCopy';
+import Icon from '~shared/Icon';
+// Note that we use CSS here because MaskedAddress css file will override font-normal by tailwind due to injection order
+import styles from './CopyableAddressV2.css';
+
+interface Props {
+  /** Address to display */
+  children: Address;
+  /** Indicates that the full address should be shown instead of an abbreviated one */
+  full?: boolean;
+  /** In some occasions we want to show the button to copy only */
+  hideAddress?: boolean;
```

Comment:
I don't see this component with this variant being used anywhere. And if it is needed indeed, the button to copy address should be an independent component that this one depends on, not the other way around